connectors/kafka/element-templates/kafka-outbound-connector-hybrid.json

```diff
@@ -174,8 +174,11 @@
   }, {
     "id" : "avro.schema",
     "label" : "Avro schema",
-    "description" : "Optional Avro schema for the message value",
-    "optional" : true,
+    "description" : "Avro schema for the message value",
+    "optional" : false,
```

Comment:
isn't `false` the default value? If so, we might remove this line I guess

Reply:
Ha ok, I resolve the conv then sorry about that

Reply:
Great feedback @johnBgood 
@Oleksiivanov Thanks for the clarification. Can you please add this feedback to this ticket please: https://github.com/camunda/connectors/issues/1797